msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -72,6 +72,11 @@ Meetings enable collaboration, partnership, informed communication, and shared f
 
 You must be familiar with the following product concepts to create custom meeting experiences with apps in Microsoft Teams.
 
+> [!IMPORTANT]
+>
+> * Currently, third-party and meeting line-of-business apps are available in Government Community Cloud (GCC) but aren't available for GCC-High and Department of Defense (DOD) tenants.
```

Comment:
maybe "third-party and line-of-business apps built for meetings and calls are available...